clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -3268,6 +3287,14 @@ private long getLatestLeaderOffsetAndHybridTopicOffset(
         : cachedPubSubMetadataGetter
             .getOffset(getTopicManager(sourceRealTimeTopicKafkaURL), leaderTopic, partitionToGetLatestOffsetFor);
 
+    if (lastOffsetInRealTimeTopic < 0) {
+      LOGGER.warn(
```

Comment:
can we rate limit this logging?

Reply:
We can... but IDK if we should 🤔 ... maybe it should be a metric instead 🤷 ?

Reply:
Actually there is already a metric for that called `getLatestOffsetIsAbsent` which is emitted from the `KafkaConsumerService`, called just above. So I guess it's fine to rate limit this log.

Reply:
No that's not correct, actually... the `getLatestOffsetIsAbsent` will not be called in cases where get into this if block. I may want another metric for that...